framework/PageForm/PageForm.tsx

```diff
@@ -133,12 +135,12 @@ export function PageForm<T extends object>(props: {
         {error && (
           <Alert
             variant="danger"
-            title={error.message ?? ''}
+            title={t('Error')}
```

Comment:
is there no support for message title and message description now?

Reply:
@himdel I can add a conditional to add this one, but since error now is a string, or list of string messages, I refactored that one.

Reply:
I moved the Alert to its own component now, allowing different titles and components, based on the type of error.